packages/content-management/content_insights/content_insights_public/src/components/activity_view.tsx

```diff
@@ -97,10 +98,7 @@ export const ActivityView = ({ item }: ActivityViewProps) => {
   );
 };
 
-const dateFormatter = new Intl.DateTimeFormat(i18n.getLocale(), {
-  dateStyle: 'long',
-  timeStyle: 'short',
-});
+const dateFormatter = (when: string) => moment(when).format('MMMM D, YYYY [at] h:mm A');
```

Comment:
looks good, I suggest we use [a localized format](https://momentjs.com/docs/#/displaying/format/) instead of hardcoded one. `LLL` looks the most similar to what we had.

Reply:
It seems that by changing to use 'LLL' we will lose "at" that is present in the current implementation.
How about the latest change?
This is a follow up on our idea to combine the current solution with only importing timeZone from the 'moment' library.